testhelper/ignore_err.go

```diff
@@ -0,0 +1,6 @@
+package testhelper
+
+// IgnoreErr is a helper function to ignore errors in tests, useful when closing bodies for example
+func IgnoreErr(f func() error) {
```

Comment:
we do ignore error at many other places apart from test. So, making it accessible globally would make sense, maybe in `utils/misc`.

Reply:
I'll just get rid of it due to linting issues: https://github.com/rudderlabs/rudder-server/pull/2411#discussion_r967192780